packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -650,7 +652,14 @@ class TextSelectionGestureDetector extends StatefulWidget {
   /// Called for a single long tap that's sustained for longer than
   /// [kLongPressTimeout] but not necessarily lifted. Not called for a
   /// double-tap-hold, which calls [onDoubleTapDown] instead.
-  final GestureLongPressCallback onSingleLongTapDown;
+  final GestureLongPressDragStartCallback onSingleLongTapStart;
+
+  /// Called for a single long tap that's dragged while the pointer is still
+  /// down.
+  final GestureLongPressDragUpdateCallback onSingleLongTapDragUpdate;
+
+  /// Called for a single long tap that's now lifted.
```

Comment:
Called after [onSingleLongTapStart] when the pointer is lifted.

Reply:
thanks